build-android/app/src/main/java/com/google/firebase/codelab/friendlychat/MainActivity.kt

```diff
@@ -57,6 +56,16 @@ class MainActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        // When running in debug mode, connect to the Firebase Emulator Suite
+        // "10.0.2.2" is a special value which allows the Android emulator to
+        // connect to "localhost" on the host computer. The port values are
+        // defined in the firebase.json file.
+        if (BuildConfig.DEBUG) {
```

Comment:
is this BuildConfig.DEBUG from the AuthUI package? I had trouble with getting email/password auth to work for a bit and I'm wondering if I imported the wrong BuildConfig.DEBUG

Reply:
No it should be `BuildConfig` from `com.google.firebase.codelab.friendlychat` ... let me see if I can add an import.